modules/src/em_code/em.c

```diff
@@ -69,6 +72,7 @@ C_flush() {
 		return;
 	}
 #endif
+	count = (int)(C_opp - obuf);
```

Comment:
What's this variable for? Was it supposed to be used in sys_write() below?

Reply:
Legacy code when debugging. It has been removed.